WordPress/WordPressUITests/Tests/EditorTests.swift

```diff
@@ -19,18 +19,22 @@ class EditorTests: XCTestCase {
     }
 
     override func tearDown() {
-        _ = editorScreen.goBack()
+        if editorScreen.isLoaded() {
+            _ = editorScreen.goBack()
+        }
         LoginFlow.logoutIfNeeded()
         super.tearDown()
     }
 
     func testSimplePublish() {
+        let title = "Hello XCUI World!"
         let longText = String(repeating: "very ", count: 20) + "long text in a galaxy not so far away"
```

Comment:
🤔An idea that came to mind was maybe in the future we could have the data for editor tests outside of the code to have the option to dynamically test varying content

Reply:
Absolutely! I was thinking about adding a data helper to handle any arbitrary text input in the tests, as well as review exactly what to include in the basic post publishing test (possibly add e.g. some amount of text formatting and/or an image).